src/codegen/bindgen-lib.ts

```diff
@@ -1,5 +1,9 @@
-// This is the public API for `bind.ts` files
-// It is aliased as `import {} from 'bindgen'`
+/**
```

Comment:
does jsdoc make this a comment for the file or the next declared symbol (Type)? i could be totally wrong and this form is correct.

Reply:
for `//`? no. It _should_ make one for `/** */` but I'm not seeing it. Maybe there's a problem with the tsconfig?